root/release/edit/tracklist.tt

```diff
@@ -338,6 +338,23 @@
        %]
     </table>
 
+    <!-- ko if: hasUselessMediumTitle -->
```

Comment:
IIRC @mwiencek is currently working on removing knockout dependency, please make sure there is no conflict or do arrange to prioritize one change over the other.

Reply:
I expect it's easy to change this one when changing all the others, but I'd also be happy to change this if that code got merged earlier. Whatever I can do to get rid of knockout :D